app/page-partials/my-dashboard/IntegrationList.tsx

```diff
@@ -237,13 +237,14 @@ export default function IntegrationList({ setIntegration, setIntegrationCount, s
                 <td>{integration.projectName}</td>
                 <td>{getStatusDisplayName(integration.status || 'draft')}</td>
                 <td>{integration.serviceType === 'gold' ? 'Gold' : 'Silver'}</td>
-                <td>
+                <td style={{ float: 'right', marginTop: '10px' }}>
                   <ActionButtons
                     request={integration}
                     onDelete={() => {
                       loadIntegrations();
                     }}
                     defaultActiveColor="#fff"
+                    delIconStyle={{ marginLeft: '7px' }}
```

Comment:
hm... does it okay in small screen sizes?

Reply:
the display works fine with different window sizes, no overlap or weird situations